recipes/pref_align_scan/README.md

```diff
@@ -0,0 +1,58 @@
+# Ablating Preference Alignment Algorithms
+
+This directory contains various ablations for three algorithms: DPO, IPO, and KTO. Each algorithm has been subjected to different ablations to study their performance and behavior under different conditions. Two different models and datasets have been used to ablated the performance of each algorithm
```

Comment:
> Two different models and datasets have been used to **ablated** the performance of each algorithm

Should ablated potentially be evaluate or another term? I apologize if it should be ablated as is, I'm not that familiar with the terminology.

Reply:
agree